gatling-core/src/main/scala/io/gatling/core/runner/Runner.scala

```diff
@@ -47,6 +48,8 @@ class Runner(selection: Selection)(implicit configuration: GatlingConfiguration)
       GatlingActorSystem.start()
       val simulation = simulationClass.newInstance
 
+      if (simulation.isInstanceOf[GatlingFunSpec]) simulation.asInstanceOf[GatlingFunSpec].setupRegisteredSpecs
```

Comment:
I thought of pattern matching, but I figured it was 3 lines of code where one will do. 
But I'll go change it since pattern matching is more scala-y anyway :)